### DISCUSSION TITLE
Possibility to set recognize options in config.php?

### DISCUSSION THREAD

**Discussion by MartiniMoe** [nextcloud/recognize#1173] [Q&A] [answered]
Hi there,

is it possible to set recognize options in Nextclouds config.php instead of manually setting them in the settings page?
For example to set the NodeJS binary path it would be cool to be able to put something like 
```
'recognize' =>
array(
  'node_binary_path' => '/path/to/node_binary',
),
```
in the config.php.

If this somehow is already possible, where can I find the corresponding config key names?

**Comment by marcelklehr** [comment, marked as answer]
Hey, sorry, that is not possible. You can however set the values using the command line interface `occ config:app:get` and `occ config:app:set`
